include/rapidcheck/detail/Unicode.hpp

```diff
@@ -0,0 +1,96 @@
+#pragma once
+
+namespace rc {
+namespace detail {
+
+template<typename T, typename RandomType>
```

Comment:
The current extended ASCII string generator does not take size into account for individual characters but I think it should. So this one might just as well be designed to do so from the get go.